content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -131,19 +131,27 @@ NormalizeScore extension point.
 
 ### Reserve {#reserve}
 
-This is an informational extension point. Plugins which maintain runtime state
-(aka "stateful plugins") should use this extension point to be notified by the
-scheduler when resources on a node are being reserved for a given Pod. This
-happens before the scheduler actually binds the Pod to the designated node, and
-it exists to prevent race conditions while the scheduler waits for the bind to
-succeed. This is the penultimate step of the scheduling cycle.
-
-Each enabled reserve plugin may individually succeed or fail. If a reserve
-plugin fails, subsequent reserve plugins are not executed. All plugins must
-succeed for the extension point to succeed. If the extension point succeeds, the
-rest of the scheduling cycle and the binding cycle are executed. If the
-extension point fails due to one or more plugins failing, the
-[Unreserve](#unreserve) extension point is triggered.
+A reserve plugin implements two methods, namely Reserve and Unreserve, to serve
+as informational extension points. Plugins which maintain runtime state (aka
+"stateful plugins") should use these extension points to be notified by the
+scheduler when resources on a node are being reserved and unreserved for a given
+Pod.
+
+The Reserve extension point, which is the penultimate step of the scheduling
+cycle, is triggered before the scheduler actually binds the Pod to the
```

Comment:
```suggestion
The Reserve phase happens before the scheduler actually binds the Pod to the
```

I think there is no need to say penultimate, given that it can be deduced from the description of `Permit`.

Reply:
Done.